src/VssCommandProcessor.cpp

```diff
@@ -50,7 +50,6 @@ VssCommandProcessor::VssCommandProcessor(
   subHandler = subhandler;
   // TODO: add accessValidator as dependency
   accessValidator = std::make_shared<AccessChecker>(tokenValidator);
-  requestValidator = new VSSRequestValidator(logger);
```

Comment:
Where is it gone to? It is needed for verifying gen2 requests (Removing this should break all get requests)